Apps/Common/src/Factories/RequestResultFactory.cs

```diff
@@ -0,0 +1,116 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+
+namespace HealthGateway.Common.Factories
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using HealthGateway.Common.Data.ErrorHandling;
+    using HealthGateway.Common.Data.ViewModels;
+    using HealthGateway.Common.ErrorHandling;
+
+    /// <summary>
+    /// Factory for  <see cref="RequestResult{T}"/> instances.
+    /// </summary>
+    public static class RequestResultFactory
```

Comment:
Are there unit tests for this static factory?  Will this affect code coverage if there aren't any unit tests?

Reply:
There is no specific unit tests to cover the factory methods, however they're covered in the services that use these methods. There's little value in testing these methods as they're merely construction shortcuts without logic worth testing,

Reply:
fine